hash2curve/src/hash2field/expand_msg/xmd.rs

```diff
@@ -146,12 +147,40 @@ where
     }
 }
 
+/// Error type for [`ExpandMsgXmd`].
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ExpandMsgXmdError {
+    /// The domain separation tag is empty.
+    DSTError(EmptyDST),
+    /// The length in bytes is too large.
+    ///
+    /// `len_in_bytes` must be at most `255 * HashT::OutputSize`
+    OversizedLen,
```

Comment:
We should avoid using the term "oversized" here, its a bit inflated in hash2curve with "oversized DST".
```suggestion
    Length,
```
This seems enough to me, but `TooLong` is also an option. Its bikeshedding at this point.

Reply:
Fixed in https://github.com/RustCrypto/elliptic-curves/pull/1338/commits/5304eb19c9a8e92fcc8fd06a9a1531e5ea1c7215.